pisa/analysis/analysis.py

```diff
@@ -3223,7 +3233,12 @@ def nofit_hypo(self, data_dist, hypo_maker, hypo_param_selections,
                     metric_val += data
                 priors = hypo_maker.params.priors_penalty(metric=metric[0]) # uses just the "first" metric for prior
                 metric_val += priors
-            else: # DistributionMaker object
+            elif isinstance(data_dist, list): # DistributionMaker object with VarBinning
```

Comment:
Is no special treatment needed for "generalized_poisson_llh"? (as in the case of the dist.maker with regular binning)